pipelines/build/common/config_regeneration.groovy

```diff
@@ -539,6 +544,7 @@ return {
             javaVersion,
             buildConfigurations,
             targetConfigurations,
+            activeNodeTimeout,
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs